metricbeat/module/jolokia/_meta/docs.asciidoc

```diff
@@ -1,8 +1,13 @@
-This is the Jolokia module.
+This module collects metrics from
+https://jolokia.org/reference/html/agents.html[Jolokia agents] running on a
+target JMX server or dedicated proxy server. The default metricset is `jmx`.
+
+To collect metrics, {beatname_uc} communicates with a Jolokia HTTP/REST
```

Comment:
Took some of this language from https://github.com/elastic/beats/issues/4344, but I found some of the language used there to be a bit redundant.